.github/workflows/e2e-test-setup.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: e2e-test-setup
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  e2e-test-setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # E2E test cases on Notation: Setup
+      - name: Setup Notation
+        uses: notaryproject/notation-action/setup@main
```

Comment:
This is not right when this workflow is triggered by a PR.
It should be `./setup`

Reply:
Same applies to other places.

Reply:
Oh yeah, nice catch!